js/_hole-common.tsx

```diff
@@ -198,7 +198,7 @@ export function setState(code: string, editor: EditorView) {
                 extensions[lang as keyof typeof extensions] ?? [],
                 // These languages shouldn't match brackets.
                 ['fish', 'hexagony'].includes(lang)
-                    ? [] : extensions.bracketMatching,
+                    ? [extensions.zeroIndexedLineNumbers] : [extensions.lineNumbers, extensions.bracketMatching],
```

Comment:
Nitpicking, but `extensions.zeroIndexedLineNumbers` probably doesn't need to be wrapped in an array.